File: swagger/models/assigned_client_index.py
from .base_model import BaseModel


class AssignedClientIndex(BaseModel):
	"""
	Represents a client index value assigned to a client

	Attributes:

		id
		The index ID assigned to the client.

		value_id
		The index’s value ID.
	"""

	input_map = {
		'Id': 'id',
		'ValueId': 'value_id',
		}

	output_map = {
		'id': 'Id',
		'value_id': 'ValueId',
		}

	id = None
	value_id = None

File: swagger/Models/AssignedClientIndex.php
<?php

namespace App\Mindbody\Models;

/**
 * Represents a client index value assigned to a client
 */
class AssignedClientIndex extends BaseModel
{
	/**
	 * The index ID assigned to the client.
	 * 
	 * @var integer format:int32
	 */
	public $id;
	/**
	 * The index’s value ID.
	 * 
	 * @var integer format:int32
	 */
	public $valueId;

	protected function getInputMap()
	{
		return [
			'Id' => ['id', 'integer', 'int32'],
			'ValueId' => ['valueId', 'integer', 'int32'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'id' => ['Id', 'integer', 'int32'],
			'valueId' => ['ValueId', 'integer', 'int32'],
			];
	}
}
